mmhuman3d/utils/ffmpeg_utils.py

```diff
@@ -430,7 +431,8 @@ def images_to_array(
 
     num_frames = len(os.listdir(input_folder))
     start = (min(start, num_frames - 1) + num_frames) % num_frames
-    end = (min(end, num_frames - 1) + num_frames) % num_frames
+    end = (min(end, num_frames - 1) +
+           num_frames) % num_frames + 1 if end is not None else num_frames
```

Comment:
Similar as above, both `start` and `end`.

Reply:
this is still useful since in `ffmpeg`, `start_frame` and `end_frame` must be None.